rework_experiments/lib/navigation/wolt_modal_sheet.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:rework_experiments/navigation/wolt_modal_sheet_coordinator.dart';
+import 'package:rework_experiments/navigation/wolt_modal_sheet_path.dart';
+import 'package:rework_experiments/navigation/wolt_modal_sheet_route.dart';
+import 'package:rework_experiments/navigation/wolt_navigator.dart';
+
+/// Main widget to show [WoltModalSheet] and use [WoltNavigator].
+/// It is important to pass all supported pages that can be displayed in the [WoltModalSheet].
+class WoltModalSheet extends StatefulWidget {
+  final List<WoltModalSheetRoute> supportedPages;
+  final List<WoltModalSheetPath> initialPage;
+
+  const WoltModalSheet({
+    super.key,
+    required this.supportedPages,
+    required this.initialPage,
```

Comment:
why is the initial page required? Can we make it optional and assume the first WoltModalSheetRoute is the initial?